### DISCUSSION TITLE
[Proposal]: Direct Storage integration

### DISCUSSION THREAD

**Discussion by Redageddon** [dotnet/runtime#67324] [General]
Like the title says, It would be nice to utilize Direct Storage as a C# feature. I am aware that it just recently came out and is still in its infancy, but I just thought it would be nice to add as a proposal to the language.

**Comment by CyrusNajmabadi** [comment]
In general, this would not likely ever be a language feature but rather an API.  

If you did want a language change, it would need to be more specific as to what you were looking for.

**Comment by tannergooding** [comment]
DirectStorage is Windows and hardware specific. I couldn't see the BCL adding direct/exposed support for it.
